nvflare/fuel/f3/cellnet/connector_manager.py

```diff
@@ -133,9 +133,10 @@ def is_adhoc_allowed(self, c1: FqcnInfo, c2: FqcnInfo) -> bool:
             return False
 
         # we only allow gen2 (or above) cells to directly connect
-        if c1.gen >= 2 and c2.gen >= 2:
-            return True
-        return False
+        # if c1.gen >= 2 and c2.gen >= 2:
+        #     return True
+        # return False
+        return True
```

Comment:
We need to allow this because the NDAS cell is gen 1. 
We disallowed this before because we thought only CJs need to have ad-hoc conns (e.g. for site-to-site). But now we know that assumption isn't right.
The ad-hoc connection is just an idea that didn't have a real-world use case until now. So we need to adjust it to work for the real usage.

Reply:
Got it, thanks for the explanation